.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -96,10 +96,10 @@ jobs:
           ARTEFACT_REGISTRY_USERNAME: ${{ github.actor }}
           ARTEFACT_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
           TARGET_BRANCH: ${{ inputs.target-branch }}
-          RELEASE: ${{ if inputs.stage == 'release' && 'yes' || 'no' }}
+          RELEASE: ${{ inputs.stage == 'release' && 'yes' || 'no' }}
```

Comment:
Hi @stevenhorsman !

Sorry, I reviewed this and completely overlooked all those mistakes! I should take some vacation...oh wait...

Seriously now, let's enable this https://github.com/marketplace/actions/actionlint action as part of the static-checks to lint the workflow files. Probably some on the mistakes here we would have caught.

Reply:
I tried running actionlint locally and (probably due to user error) it didn't detect this issues. I think it's worth a try anyway

Reply:
hmmm Interesting